compiler/src/prog.ml

```diff
@@ -463,8 +463,10 @@ let rec has_call_or_syscall_i i =
 
 and has_call_or_syscall c = List.exists has_call_or_syscall_i c
 
-let has_annot a { i_annot ; _ } =
-  List.exists (fun (k, _) -> String.equal (L.unloc k) a) i_annot
+let has_annot a { i_annot } = Annotations.has_symbol a i_annot
```

Comment:
Please keep the `; _` part (or use the dot syntax, as in `x.i_annot`).

Reply:
Oh, sorry. What does this mean without the `; _` then? I can't find it in the documentation

Reply:
With an explicit trailing underscore, it means: “other fields are to be ignored, even if more fields are added to this record type in the future”. It is analogous to a final `| _ -> …` catch-all branch in a `match` construct.

Reply:
Done